lib/matplotlib/lines.py

```diff
@@ -1250,9 +1244,6 @@ def set_dashes(self, seq):
         else:
             self.set_linestyle((0, seq))
 
-    def _draw_lines(self, renderer, gc, path, trans):
```

Comment:
The entire `drawStyles` dict is now unused, and only left for backcompatibility purposes (mostly to have a *list* of drawstyles available as `matplotlib.lines.drawStyles`).  With this change none of the values actually refer to a method name anymore.  I killed the other entries in #6497 for another purpose.